Microsoft.Toolkit.Uwp/Helpers/PrintHelper/PrintHelper.cs

```diff
@@ -254,16 +254,16 @@ private void PrintTaskRequested(PrintManager sender, PrintTaskRequestedEventArgs
                 printTask.Completed += async (s, args) =>
                 {
                     // Notify the user when the print operation fails.
-                    await CoreApplication.MainView.CoreWindow.Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, () =>
+                    await CoreApplication.MainView.CoreWindow.Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, async () =>
                     {
                         foreach (var element in _stateBags.Keys)
                         {
                             _stateBags[element].Restore(element);
                         }
                         _stateBags.Clear();
-
-                        DetachCanvas();
+                        
```

Comment:
Something odd here?

Helpers\PrintHelper\PrintHelper.cs(264,1): error SA1028: Code must not contain trailing whitespace [D:\a\1\s\Microsoft.Toolkit.Uwp\Microsoft.Toolkit.Uwp.csproj]

Reply:
An unnecessary space. 😄